src/optimize.rs

```diff
@@ -246,6 +252,42 @@ fn get_self_cost(egraph: &EGraph<Mdl, TensorAnalysis>, enode: &Mdl) -> f32 {
             }
         }
 
+        Mdl::Split([_axis, _inpt]) => {
+            // Check types
+            let _axis_data = x(_axis);
+            let _inpt_data = x(_inpt);
+            assert!(_axis_data.dtype == DataKind::Scalar);
+            assert!(_inpt_data.dtype == DataKind::Tnsr);
+
+            // Get arguments
+            let t_inpt = _inpt_data.meta;
+            let axis = _axis_data.val;
+            unsafe {
+                // Get op
+                let op = (*g.model).get_or_create_split1(t_inpt, axis, 2);
```

Comment:
get_or_create_split1 suggests that there is also get_or_create_split2 or more? What the difference between the two?

Reply:
These are ffi functions generated automatically by bindgen. There are get_or_create_split and get_or_create_split1, which corresponds to the two get_or_create_split functions on the C++ side with different argument types. These ffi functions are defined in bindings.rs, which is generated during building.